rule-packs/aws-privilege-escalation.json

```diff
@@ -0,0 +1,194 @@
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "CRITICAL"
+  },
+  {
+    "name": "aws-ability-to-add-a-user-to-a-group",
+    "description": "Users with the iam:AddUserToGroup permission can use it to add themselves to an existing IAM Group in the AWS account.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND User\n  (THAT HAS aws_iam_group)?\n  (THAT ASSIGNED aws_iam_role)?\n  (THAT TRUSTS << aws_iam_role)?\n  THAT ASSIGNED AccessPolicy\n  THAT ALLOWS as perm aws_iam\nWHERE perm.normalizedActions ~= ('iam:addusertogroup' OR 'iam:*')",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "HIGH"
+  },
+  {
+    "name": "aws-ability-to-update-the-assumerolepolicydocument-of-a-role",
+    "description": "Users with the iam:UpdateAssumeRolePolicy and sts:AssumeRole permissions would be able to change the assume role policy document of any existing role to allow them to assume that role.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND User\n  (THAT HAS aws_iam_group)?\n  (THAT ASSIGNED aws_iam_role)?\n  (THAT TRUSTS << aws_iam_role)?\n  THAT ASSIGNED AccessPolicy\n  THAT ALLOWS as perm aws_iam\nWHERE perm.normalizedActions ~= ('iam:updateassumerolepolicy' OR 'iam:*')\n  AND perm.normalizedActions ~= ('sts:assumerole' OR 'sts:*')",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "CRITICAL"
+  },
+  {
+    "name": "aws-ability-to-update-the-code-of-an-existing-lambda-function",
+    "description": "A user with the lambda:UpdateFunctionCode permission could update the code in an existing Lambda function with an IAM role attached so that it would import the relevant AWS library in that programming language and use it to perform actions on behalf of that role.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND User\n  (THAT HAS aws_iam_group)?\n  (THAT ASSIGNED aws_iam_role)?\n  (THAT TRUSTS << aws_iam_role)?\n  THAT ASSIGNED AccessPolicy\n  THAT ALLOWS as perm aws_lambda\nWHERE perm.normalizedActions ~= ('lambda:updatefunctioncode' OR 'lambda:*')",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "CRITICAL"
+  },
+  {
+    "name": "aws-ability-to-update-an-existing-glue-dev-endpoint",
+    "description": "A user with the glue:UpdateDevEndpoint permission would be able to update the associated SSH public key of an existing Glue development endpoint, to then SSH into it and have access to the permissions the attached role has access to.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND User\n  (THAT HAS aws_iam_group)?\n  (THAT ASSIGNED aws_iam_role)?\n  (THAT TRUSTS << aws_iam_role)?\n  THAT ASSIGNED AccessPolicy\n  THAT ALLOWS as perm * aws_glue\nWHERE perm.normalizedActions ~= ('glue:updatedevendpoint' OR 'glue:*')",
```

Comment:
Typo in query, need to remove the `*` in `THAT ALLOWS as perm * aws_glue`.

That line should be `THAT ALLOWS as perm aws_glue`